src/shared/components/event-start/event-guard/event-start.guard.ts

```diff
@@ -17,8 +17,8 @@ export class EventStartGuard implements CanActivate {
   }
 
   public canActivate(route: ActivatedRouteSnapshot): Observable<boolean> {
-    // Checks must be performed only for Work Allocation 2
-    if (this.appConfig.getWorkAllocationApiUrl().toLowerCase() === 'workallocation2') {
+    // Checks must be performed only for Work Allocation 
+    if (this.appConfig.getWorkAllocationApiUrl().toLowerCase() === 'workallocation') {
```

Comment:
Do we even need this check now if we're removing the difference between 1 and 2? We always need workallocation available as far as I'm aware